google-cloud-storage/src/test/java/com/google/cloud/storage/ITGrpcStorageImplUploadRetryTest.java

```diff
@@ -241,12 +241,12 @@ private static final class Resumable {
         StartResumableWriteResponse.newBuilder().setUploadId(uploadId).build();
 
     private static final ChecksummedData checksummedData =
-        TestUtils.getChecksummedData(ByteString.copyFrom(bytes), Hasher.enabled());
+        TestUtils.getChecksummedData(ByteString.copyFrom(bytes), Hasher.noop());
     private static final WriteObjectRequest req1 =
         WriteObjectRequest.newBuilder()
             .setUploadId(uploadId)
             .setChecksummedData(checksummedData)
-            .setObjectChecksums(ObjectChecksums.newBuilder().setCrc32C(checksummedData.getCrc32C()))
+            // .setObjectChecksums(ObjectChecksums.newBuilder().setCrc32C(checksummedData.getCrc32C()))
```

Comment:
is this meant to be commented out or completely removed?

Reply:
completely removed, will update.